src/transformers/convert_slow_tokenizer.py

```diff
@@ -429,6 +430,12 @@ def __init__(self, *args):
             m.ParseFromString(f.read())
         self.proto = m
 
+        if self.proto.trainer_spec.byte_fallback:
```

Comment:
Without the `src/transformers/utils/sentencepiece_model_pb2.py` file update, the `self.proto.trainer_spec.byte_fallback` call would have returned a `AttributeError` with the message "byte_fallback"